app/src/main/java/uk/gov/onelogin/mainnav/ui/MainNavScreen.kt

```diff
@@ -96,7 +96,7 @@ fun MainNavScreen(
                 HomeScreen(openDeveloperPanel = openDeveloperPanel)
             }
             composable(BottomNavDestination.Wallet.key) {
-                walletScreenViewModel.walletSdk.WalletApp(deeplink = "", adminEnabled = true)
+                walletScreenViewModel.walletSdk.WalletApp(deeplink = "", adminEnabled = false)
```

Comment:
_I know this isn't something you have done_
The naming convention for booleans is to prepend them with `is`, `has`, or `should`, so they read grammatically correct when used in conditionals or as predicates. The same is true for functions that return booleans, as well as boolean variables.